spec/v2/index.bs

```diff
@@ -16,6 +16,9 @@ Local Boilerplate: header yes
 Metadata Include: This version off
 </pre>
 
+Advisement: This document is a work in progress and should not be used for Conformance Testing. Please refer to the [latest
+stable version of the Technical Specifications]([LATEST]) for that end. All feedback is welcomed.
+
```

Comment:
I know this is a very nit-picky comment, but "for that end" isn't a very formal expression in English and given the Tech Specs are quite a formal document, I would advise a different phrasing. Perhaps "Please refer to the latest stable version of the Technical Specifications for conformance testing. All feedback is welcome!" or something like that.

Reply:
Thank you! I was trying to avoid repetition, but I agree that it is better than sounding less formal.